src/autogpt_plugins/news_search/test_auto_gpt_news_search.py

```diff
@@ -0,0 +1,40 @@
+import os
+import requests
+import unittest
+from typing import List
+from . import AutoGPTNewsSearch
+from .news_search import news_search
```

Comment:
Fix your import from class-ifying the file

Reply:
Done